config/locales/es.yml

```diff
@@ -0,0 +1,632 @@
+  registration_finished_login: "You can change or update your enterprise at any stage by logging into Open Food Network and going to Admin."
+  registration_finished_activate: "Activate %{enterprise}."
+  registration_finished_activate_instruction_html: "We've sent a confirmation email to <strong>%{email}</strong> if it hasn't been activated before.<br/>
+Please follow the instructions there to make your enterprise visible on the Open Food Network."
+  registration_finished_action: "Open Food Network home"
+  registration_type_headline: "Last step to add %{enterprise}!"
+  registration_type_question: "Are you a producer?"
+  registration_type_producer: "Yes, I'm a producer"
+  registration_type_no_producer: "No, I'm not a producer"
+  registration_type_error: "Please choose one. Are you are producer?"
+  registration_type_producer_help: "Producers make yummy things to eat &/or drink. You're a producer if you grow it, raise it, brew it, bake it, ferment it, milk it or mould it."
+  registration_type_no_producer_help: "If you’re not a producer, you’re probably someone who sells and distributes food. You might be a hub, coop, buying group, retailer, wholesaler or other."
+  create_profile: "Create Profile"
+  registration_images_headline: "Thanks!"
+  registration_images_description: "Let's upload some pretty pictures so your profile looks great! :)"
+  registration_detail_headline: "Let's Get Started"
+  registration_detail_enterprise: "Woot! First we need to know a little bit about your enterprise:"
+  registration_detail_producer: "Woot! First we need to know a little bit about your farm:"
+  registration_detail_name_enterprise: "Enterprise Name:"
+  registration_detail_name_producer: "Farm Name:"
+  registration_detail_name_placeholder: "e.g. Charlie's Awesome Farm"
+  registration_detail_name_error: "Please choose a unique name for your enterprise"
+  registration_detail_address1: "Address line 1:"
+  registration_detail_address1_placeholder: "e.g. 123 Cranberry Drive"
+  registration_detail_address1_error: "Ingrese una dirección"
+  registration_detail_address2: "Linea de dirección 2:"
+  registration_detail_suburb: "Barrio:"
+  registration_detail_suburb_placeholder: "e.g. Northcote"
+  registration_detail_suburb_error: "Ingrese el barrio"
+  registration_detail_postcode: "Código postal:"
+  registration_detail_postcode_placeholder: "e.g. 3070"
+  registration_detail_postcode_error: "Código postal requerido"
+  registration_detail_state: "Estado:"
+  registration_detail_state_error: "Estado requerido"
+  registration_detail_country: "País:"
+  registration_detail_country_error: "Seleccione un país"
+  fees: "Tarifas"
+  item_cost: "Costo del artículo"
+  bulk: "Agrupar"
+  shop_variant_quantity_min: "mínimo"
+  shop_variant_quantity_max: "máximo"
+  contact: "Contacto"
+  follow: "Seguir"
+  shop_for_products_html: "Comprar <span class=\"turquoise\">%{enterprise}</span> productos en:"
```

Comment:
I think this one is wrong.
"Comprar productos de <span class=\"turquoise\">%{enterprise}</span> en:"